nifi-extension-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/HandleHttpRequest.java

```diff
@@ -380,7 +388,19 @@ synchronized void initializeServer(final ProcessContext context) throws Exceptio
         final String clientAuthValue = context.getProperty(CLIENT_AUTH).getValue();
         final Server server = createServer(context);
 
-        final StandardServerConnectorFactory serverConnectorFactory = new StandardServerConnectorFactory(server, port);
+        NiFiProperties nifiProperties = NiFiProperties.createBasicNiFiProperties(null);
+        int requestMaxHeaderSize = DataUnit.parseDataSize(nifiProperties.getWebMaxHeaderSize(), DataUnit.B).intValue();
```

Comment:
Following the pattern of the rest of the variables these also should be declared `final`
```suggestion
        final NiFiProperties nifiProperties = NiFiProperties.createBasicNiFiProperties(null);
        final int requestMaxHeaderSize = DataUnit.parseDataSize(nifiProperties.getWebMaxHeaderSize(), DataUnit.B).intValue();
```

Reply:
lines removed by new commit